modules/tinymce/changelog.txt

```diff
@@ -1,4 +1,5 @@
 Version 5.6.0 (TBD)
+    Added support for `image_file_types` setting in image file uploader to determine which image file extensions are valid for upload #TINY-6224
```

Comment:
Should this be below the next changelog entry? It feels a bit weird saying we added support for something and then on the next line saying we added the new setting.

Also, I think there's a `the` missing between `in image`?